README.md

```diff
@@ -131,6 +131,19 @@ interesting are:
   $ cd -
   $ grunt alloy-css
   ```
+  
+* To test updating custom AlloyEditor skin against custom platform-ui-assets-bundle builds:
+
+  ```bash
+  $ composer install --prefer-sopurce
+  $ cd vendor/ezsystems/platform-ui-assets-bundle/
+  $ git checkout <your_branch>
+  $ ./bin/prepare_release.sh -v X.Y.X
+  $ git checkout vX.Y.X
+  $ echo -e `cd Resources/public/vendors/alloy-editor/ && npm install`
```

Comment:
lol thanks for the new title ;)
oh yeah the `cd -` that's a good reason, then that's fine